diff --git a/node_modules/@hcaptcha/react-native-hcaptcha/Hcaptcha.js b/node_modules/@hcaptcha/react-native-hcaptcha/Hcaptcha.js
index 2936b4c..2966a65 100644
--- a/node_modules/@hcaptcha/react-native-hcaptcha/Hcaptcha.js
+++ b/node_modules/@hcaptcha/react-native-hcaptcha/Hcaptcha.js
@@ -506,6 +506,7 @@ const styles = StyleSheet.create({
   webview: {
     backgroundColor: 'transparent',
     width: '100%',
+    flex: 1,
   },
 });
 
